docs/README.rst

```diff
@@ -1016,13 +1036,15 @@ functionality for particular interfaces.
 
    >>> @zope.interface.implementer(ICustomAdapt)
    ... class CustomAdapt(object):
-   ...    pass
+   ...     pass
    >>> ICustomAdapt('a string')
    'a string'
    >>> ICustomAdapt(CustomAdapt())
    <CustomAdapt object at ...>
 
-.. seealso:: :func:`zope.interface.interfacemethod`
+.. seealso:: :func:`zope.interface.interfacemethod`, which explains
+   how to override functions in interface definitions and why, prior
+   to Python 3.6, the zero-argument version of `super` cannot be used.
```

Comment:
Will this autolink all the way to the Python standard library reference?  Or did you accidentally use the Markdown version of preformatted text with single backticks instead of ReSTs double backticks?